code/jobs/SESQueuedMail.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * SESQueuedMail
+ *
+ * @author Stephen McMahon <stephen@silverstripe.com.au>
+ */
+class SESQueuedMail extends AbstractQueuedJob implements QueuedJob {
```

Comment:
can you add a getSignature method that returns an md5 of the $this->Subject . json_encode($this->To) ? Otherwise it'll serialise and md5 _all_ the properties.